apollo-router/src/metrics/mod.rs

```diff
@@ -1245,6 +1260,115 @@ macro_rules! assert_histogram_not_exists {
     };
 }
 
+pub(crate) fn count_operation_error_codes(
+    codes: Vec<&str>,
```

Comment:
NIT: but if you don't need ownership let's use a ref
```suggestion
    codes: &[&str],
```